sql/core/src/test/resources/tpcds-plan-stability/approved-plans-modified/q34.sf100/explain.txt

```diff
@@ -98,15 +98,15 @@ Input [5]: [ss_customer_sk#1, ss_hdemo_sk#2, ss_store_sk#3, ss_ticket_number#4,
 Output [4]: [hd_demo_sk#11, hd_buy_potential#12, hd_dep_count#13, hd_vehicle_count#14]
 Batched: true
 Location [not included in comparison]/{warehouse_dir}/household_demographics]
-PushedFilters: [IsNotNull(hd_vehicle_count), IsNotNull(hd_dep_count), Or(EqualTo(hd_buy_potential,>10000         ),EqualTo(hd_buy_potential,Unknown        )), GreaterThan(hd_vehicle_count,0), IsNotNull(hd_demo_sk)]
```

Comment:
hmm, we optimized away `IsNotNull(hd_dep_count)`?

Reply:
This is because we remove the `elseValue` if it is null. Fixed by https://github.com/apache/spark/pull/34580/commits/26436d27384771240d7d4e12c85f11ce67cee5a8